Library/Homebrew/cmd/info.rb

```diff
@@ -62,16 +62,17 @@ def github_fork
   end
 
   def github_info f
-    if f.tap?
-      user, repo = f.tap.split("/", 2)
+    if f.path.to_s =~ HOMEBREW_TAP_DIR_REGEX
+      user, repo = $1, $2
       path = f.path.relative_path_from(HOMEBREW_LIBRARY.join("Taps", f.tap))
-    else
+      "https://github.com/#{user}/#{repo}/blob/master/#{path}"
+    elsif f.core_formula?
       user = f.path.parent.cd { github_fork }
-      repo = "homebrew"
       path = f.path.relative_path_from(HOMEBREW_REPOSITORY)
+      "https://github.com/#{user}/homebrew/blob/master/#{path}"
+    else
+      "file://#{f.path}"
```

Comment:
Doesn't it seem weird for `brew info --github` to open a local file in the browser?